src/components/AddGroup.jsx

```diff
@@ -0,0 +1,145 @@
+import React, { useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import TextField from "@material-ui/core/TextField";
+import InputLabel from "@material-ui/core/InputLabel";
+import FormHelperText from "@material-ui/core/FormHelperText";
+import FormControl from "@material-ui/core/FormControl";
+import NativeSelect from "@material-ui/core/NativeSelect";
+import Grid from "@material-ui/core/Grid";
+import Typography from "@material-ui/core/Typography";
+import Slider from "@material-ui/core/Slider";
+import Input from "@material-ui/core/Input";
+import GroupIcon from "@material-ui/icons/Group";
+import GroupAddIcon from "@material-ui/icons/GroupAdd";
+import Select from "@material-ui/core/Select";
+
+const AddGroup = () => {
+  const [groupname, setGroupname] = useState("");
+  const [groupcategory, setGroupcategory] = useState("");
+  const [groupcontent, setGroupcontent] = useState("");
+  const [groupmembernumber, setGroupmembernumber] = useState("");
+  const handleSliderChange = (e, newValue) => {
+    setGroupmembernumber(newValue);
+  };
+
+  const handleInputChange = e => {
+    setGroupmembernumber(e.target.value === "" ? "" : Number(e.target.value));
+  };
+
+  const handleBlur = () => {
+    if (groupmembernumber < 0) {
+      setGroupmembernumber(0);
+    } else if (groupmembernumber > 100) {
+      setGroupmembernumber(100);
+    }
+  };
+
+  const classes = useStyles();
+  return (
+    <div>
+      <Grid container item justify="center">
+        <div className={classes.root}>
+          <h1>
+            GROUP 추가
+            <GroupAddIcon />
+          </h1>
+          <br />
+          <div>
+            <TextField
+              id="groupname"
+              label="GROUP 이름"
+              variant="outlined"
+              value={groupname}
+              onChange={e => setGroupname(e.target.value)}
+            />
+          </div>
+          <FormControl className={classes.formControl}>
+            <InputLabel htmlFor="groupcategory">GROUP 카테고리</InputLabel>
+            <NativeSelect
+              value={groupcategory}
+              onChange={e => setGroupcategory(e.target.value)}
+              inputProps={{
+                name: "age",
+                id: "groupcategory",
+              }}
+            >
+              <option aria-label="None" value="" />
+              <option value={10}>동아리</option>
+              <option value={20}>학회</option>
+              <option value={30}>기타</option>
```

Comment:
나중에 바로 저장할 수 있도록 숫자 대신 분류 코드를 이용해주세요.

```suggestion
              <option value={"club"}>동아리</option>
              <option value={"association"}>학회</option>
              <option value={"others"}>기타</option>
```
이런식으로 해주시면 됩니다.